devops/continuous_integration/tutorial.rst

```diff
@@ -0,0 +1,141 @@
+Continuous Integration (CI) tutorial
+====================================
+
+Continuous Integration has different meanings for different users and organizations. In this tutorial we will cover the scenarios when users
+are doing changes to the source code of their packages and want to automatically build new binaries for those packages and also compute if those new package changes integrate cleanly or break the organization main products.
+
+We will use in this tutorial this small project that uses several packages (static libraries by default) to build a couple of applications, a video game and a map viewer utility:
```

Comment:
Maybe we can advance here that we will refer to those applications as _products_ later